app/controllers/fhir/r4/api_controller.rb

```diff
@@ -190,7 +195,8 @@ def update
       # Wrap updates to the CloseContact and History creation in a transaction
       ActiveRecord::Base.transaction do
         unless referenced_patient_valid_for_client?(close_contact, :patient_id) && close_contact.save(context: :api)
-          status_unprocessable_entity(format_model_validation_errors(close_contact)) && return
+          req_json = request.patch? ? patient.as_fhir.to_json : JSON.parse(request.body.string)
```

Comment:
Should be `close_contact.as_fhir.to_json` here.

Reply:
Shoot, my bad. Great catch. Fixed https://github.com/SaraAlert/SaraAlert/pull/743/commits/05c1fead5757f5a2149338f21998fc588c738e8a